src/style.css

```diff
@@ -1,3 +1,88 @@
+/* stylelint-disable font-family-name-quotes */
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
```

Comment:
```suggestion
  margin: 0;
```
no need to make body flex

Reply:
<img width="722" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/295527b1-c1de-491d-9af4-dc8b6e096f6a">